src/librustc_resolve/diagnostics.rs

```diff
@@ -1503,13 +1533,13 @@ crate fn add_missing_lifetime_specifiers_label(
                 suggest_existing(err, format!("{}<{}>", snippet, name));
             }
             (0, _, Some("&")) => {
-                suggest_new(err, "&'lifetime ".to_string());
+                suggest_new(err, "&'r ");
             }
             (0, _, Some("'_")) => {
-                suggest_new(err, "'lifetime".to_string());
+                suggest_new(err, "'r");
```

Comment:
`'r` stands for "region", which is outdated terminology. Could it be possible to go through all of the lifetime parameters in scope, and find the first unused letter?
That way it would be `'a` in many cases, which is ideal IMO.

(`'r` also shows up elsewhere in this diff, not sure how all parts relates)

Reply:
Right now we don't keep track of the existing lifetimes. I'm gonna change it to always use `'a` and we can do the _right_ thing of inspecting the available lifetimes in a follow up PR.